### PR TITLE
Add non nullability on selected tags class

### PR DESCRIPTION
**Pull request by Arthur-Milchior**

### CODE REVIEW

**Review by david-allison — APPROVED** [review]

**Comment by david-allison** [comment]
@Arthur-Milchior squash merge or rebase?

**Comment by Arthur-Milchior** [comment]
I'm fine with squash

File: AnkiDroid/src/main/java/com/ichi2/anki/dialogs/tags/TagsDialogListener.kt [hunk 1]
```diff
@@ -51,11 +51,12 @@ interface TagsDialogListener {
         @KotlinCleanup("remove the ?")
```
Comment (ShridharGoel) [inline AnkiDroid/src/main/java/com/ichi2/anki/dialogs/tags/TagsDialogListener.kt:51]: Consider changing return type to `TagsDialogListener`, then the KotlinCleanup can be removed.

Comment (ShridharGoel) [reply AnkiDroid/src/main/java/com/ichi2/anki/dialogs/tags/TagsDialogListener.kt:51]: Or do you want to have a different PR for that?

**Review by Arthur-Milchior — COMMENTED** [review]
@ShridharGoel 
Done

**Review by lukstbit — APPROVED** [review]
LGTM

**Pull request merged**